laletterbuilder/tests/test_models.py

```diff
@@ -14,6 +19,11 @@ def test_str_works_on_filled_out_models(self, db):
         decl = HabitabilityLetterFactory()
         assert str(decl) == "Boop Jones's Habitability LA Letter Builder letter"
 
+    @pytest.mark.django_db
+    def test_default_mail_choice(self, db):
+        decl = HabitabilityLetterFactory()
```

Comment:
nit: could change this to `letter =` since `decl` refers to an EFNY declaration

Reply:
In the future, if we want to implement some `clean` function on the model to check that the right combination of letter fields are filled out (e.g. if there's a fully_processed date, there's also a sent date, etc), this would be a good place to do that.